src/core/Ekubo.sol

```diff
@@ -311,6 +312,7 @@ abstract contract Ekubo is SettlerAbstract {
                             eq(sellToken, 0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee),
                             and(iszero(eq(buyToken, 0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee)), lt(sellToken, buyToken))
                         )
+                    isToken1 := iszero(zeroForOne)
```

Comment:
you can probably golf the expression for `zeroForOne` so that you directly compute `isToken1` instead